raster_analysis/tiling.py

```diff
@@ -130,10 +130,11 @@ def _execute_tiles(self) -> DataFrame:
             "environment": self.data_environment.dict(),
         }
 
-        if sys.getsizeof(json.dumps(self.raw_geom)) > LAMBDA_ASYNC_PAYLOAD_LIMIT_BYTES:
+        payload["geometry"] = self.raw_geom
+        if sys.getsizeof(json.dumps(payload)) > LAMBDA_ASYNC_PAYLOAD_LIMIT_BYTES:
+            # if payload would be too big, compress geometry
+            del payload["geometry"]
```

Comment:
i'm a sucker for one liners but don't have to use it :)
```
payload["encoded_geometry"] = encode_geometry(payload.pop("geometry)
```

Reply:
smart, I like it!